hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleResult.java

```diff
@@ -58,22 +59,23 @@ public record PreHandleResult(
 
     public PreHandleResult(
             @NonNull final PreHandleContext context,
+            @NonNull final ResponseCodeEnum status,
```

Comment:
Would it make sense to have this be `@Nullable` and use the context status if it is passed in as null?

Reply:
That may be. I'll take care of this as part of the pre-handle work